program/src/processor.rs

```diff
@@ -61,6 +61,8 @@ fn get_validator_stake_checked(
     }
 
     // Ensure the stake account belongs to the validator.
+    // Jon: I'm thinking this should be the "authority" on the stake account instead,
+    // since we don't know if all validators have access to their vote account keypair
```

Comment:
Which one?

Reply:
This guy: https://github.com/paladin-bladesmith/stake-program/blob/d26154e84a4daf32da91f2dcae47564256f56053/program/src/state/stake.rs#L37

Reply:
Done.